### PR TITLE
Gaussian noise

### PR DESCRIPTION
**Pull request by marjanfamili**
- add Gaussian blur and normalisation 
- fix plot size for smaller number of classes 
- add more columns to the 3d reconstruction output

### CODE REVIEW

**Comment by quantumjot** [comment]
Seems like a bandpass filter would be a good addition

**Comment by marjanfamili** [comment]
#141

**Comment by marjanfamili** [comment]
> Seems like a bandpass filter would be a good addition

## BandPass filter 

I am not sure why some of the high frequency features are not visible anymore ?

Screenshot 2023-06-23 at 13 40 04
screenshot-6

File: README.md [hunk 1]
```diff
@@ -167,9 +167,13 @@ this will rewrite the values for the epochs and affinity path in the config file
 At the end of the run, the code will save the final config file used for the run in the working directory. This will
 account for any changes made to the config file from the command line. Running the code again with that config file will reproduce the results.
 
+#### tools
+In the tools folder you can find notebooks which will assist you in creating the input files for Affinity-VAE or analyse teh output of the model.
```
Comment (jolaem) [inline README.md:171]: Typo 'teh'

File: README.md [hunk 2]
```diff
@@ -167,9 +167,13 @@ this will rewrite the values for the epochs and affinity path in the config file
...
 
 #### Considerations
 
 ##### Test folder : If test folder is present, the program will read the test files regardless of the eval flag
 
 ##### Evaluation: To run evaluation on a trained model you can turn the `eval` flag to True. This will load the last model present on the `states` directory (within the working directory path where you run the code) and run the evaluation on data set by the `datapath` flag. The evaluation will be saved in the `plots` and `latents` directory with the `eval` suffix on the names.
+
+##### The name of the state file is consist of avae_date_time_Epoch_latent_pose.pt
```
Comment (jolaem) [inline README.md:179]: Typo 'is consist'

File: avae/train.py [hunk 3]
```diff
@@ -44,6 +48,8 @@ def train(
     recon_fn,
     use_gpu,
     model,
+    gaussian_blur,
+    normalise,
```
Comment (jolaem) [inline avae/train.py:52]: Add both of these to the docstring

File: avae/config.py [hunk 4]
```diff
@@ -61,6 +67,9 @@
     "vis_pos": VIS_POS,
     "vis_acc": VIS_ACC,
     "vis_his": VIS_HIS,
+    "vis_sim": VIS_SIM,
     "vis_all": False,
     "model": "a",
+    "gaussian_blur": False,
+    "normalise": False,
```
Comment (jolaem) [inline avae/config.py:74]: Add these two params to the default config too

File: avae/data.py [hunk 5]
```diff
@@ -23,6 +23,8 @@ def load_data(
     eval: bool = True,
     affinity=None,
     classes=None,
+    gaussian_blur=False,
+    normalise=False,
```
Comment (jolaem) [inline avae/data.py:27]: Add to docstring

File: avae/data.py [hunk 6]
```diff
@@ -188,6 +192,8 @@ def __init__(
         amatrix=None,
         classes=None,
         transform=None,
+        gaussian_blur=False,
+        normalise=False,
```
Comment (jolaem) [inline avae/data.py:196]: Add to docstring

File: avae/evaluate.py [hunk 7]
```diff
@@ -10,15 +10,29 @@
...
+    datapath,
+    state,
+    meta,
+    lim,
+    splt,
+    batch_s,
+    collect_meta,
+    use_gpu,
+    gaussian_blur,
+    normalise,
```
Comment (jolaem) [inline avae/evaluate.py:23]: Add to docstring

File: avae/train.py [hunk 8]
```diff
@@ -339,19 +395,58 @@ def train(
...
+                x_val, np.array(y_val), mode="_valid", epoch=epoch
+            )
+
         # visualise accuracy
         if config.VIS_ACC and (epoch + 1) % config.FREQ_ACC == 0:
             train_acc, val_acc, ypred_train, ypred_val = accuracy(
                 x_train, y_train, x_val, y_val
             )
+            print(
+                "Epoch: [%d/%d] |   Gamma: %f | Beta: %f"
```
Comment (jolaem) [inline avae/train.py:441]: Additional whitespace.

File: avae/config.py (hunk 4 above)
Comment (marjanfamili) [reply avae/config.py:74]: Im not sure what you mean here. The one you have commented on, are the default values, right ?

**Pull request merged**